active/PagerControl/PagerControl.md

```diff
@@ -295,8 +310,8 @@ How often are developers using this control with an indefinite number of pages?
 
 
 ## Open Questions 
-* I need help understanding how to choose what automation control type could be used for a control like this. Can it be a combination of control types?
 * It would be cool to measure which layout view is most commonly used with the pager control, is that possible to measure using telemetry?
-
+* Is there a shortcut we can add to make it easier to move to the next and previous page? Maybe utilize the left and right bumper?
```

Comment:
NavigationView does something similiar. When in DisplayMode top, developers can opt in to let the bumpers change the selection. The API for that is: https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.shouldernavigationenabled?view=winui-2.4#Microsoft_UI_Xaml_Controls_NavigationView_ShoulderNavigationEnabled

Reply:
This would only work if you have keyboard focus on the control, which I don't think is a common use case. I think this needs to be a feature of the list control.